mk/rabbitmq-dist.mk

```diff
@@ -123,23 +163,34 @@ $(DIST_DIR)/%.ez:
 		&& grep -q '^prepare-dist::' $(SRC_DIR)/Makefile) || \
 		$(MAKE) --no-print-directory -C $(SRC_DIR) prepare-dist \
 		APP=$(APP) VSN=$(VSN) EZ_DIR=$(EZ_DIR)
-	$(verbose) (cd $(DIST_DIR) && $(ZIP) $(ZIP_V) -r $*.ez $*)
+	$(verbose) (cd $(DIST_DIR) && $(ZIP) $(ZIP_V) -r `basename $(EZ)` `basename $(EZ) .ez`)
 	$(verbose) rm -rf $(EZ_DIR)
 
+$(MIX_DIST_EZS): $(mix_task_archive_deps_installed)
+	$(verbose) rm -rf $(DIST_DIR)/tmp
+	$(verbose) mkdir $(DIST_DIR)/tmp
+	$(verbose) cd $(SRC_DIR) && MIX do deps.get, deps.compile, compile, archive.build.all -e -o $(realpath $(DIST_DIR))/tmp
+
+$(mix_task_archive_deps_installed):
+	mix archive.install --force https://github.com/hairyhum/mix_task_archive_deps/releases/download/0.1.0/mix_task_archive_deps-0.1.0.ez
+
 # We need to recurse because the top-level make instance is evaluated
 # before dependencies are downloaded.
 
 MAYBE_APPS_LIST = $(if $(shell test -f $(ERLANG_MK_TMP)/apps.log && echo OK),$(ERLANG_MK_TMP)/apps.log)
 
 dist:: $(ERLANG_MK_RECURSIVE_DEPS_LIST) all
+	$(gen_verbose) rm -rf $(DIST_DIR)/*.ez
```

Comment:
There can only be a single `$(*_verbose)` line, otherwise the target will be reported as if it was built twice. You can change the next line to `($verbose)`.